### PR TITLE
form.file() input has unwanted background colour and layout

### LINKED ISSUE #24: form.file() input has unwanted background colour and layout
* Should not have background colour
* Input should be vertically inline with label

screen shot 2016-02-03 at 14 09 00

### PR DESCRIPTION
**Pull request by jamesjacobs**
Resolves #24

### CODE REVIEW

**Comment by langy** [comment]
:+1:

File: views/pulsar/v2/helpers/form.html.twig [hunk 1]
```diff
@@ -516,6 +516,7 @@ data-*      | string | Data attributes, eg: `'data-foo': 'bar'`
 				elem.input(options
 					|exclude('class help label')
 					|defaults({'type': 'file'})
+                    |merge({'class': 'file'})
```
Comment (Stanton) [inline views/pulsar/v2/helpers/form.html.twig]: Tabbing

Comment (langy) [inline views/pulsar/v2/helpers/form.html.twig]: Weird how it broke even more after watching you fix it @jamesjacobs

Comment (Stanton) [inline views/pulsar/v2/helpers/form.html.twig]: Unit test fixtures will need updating, run 'phpunit' to see which ones broke

Comment (Stanton) [inline views/pulsar/v2/helpers/form.html.twig]: If file is hard tabs, convert entire file to spaces

**Comment by Stanton** [comment]
:+1:

**Pull request merged**